sui-execution/latest/sui-adapter/src/temporary_store.rs

```diff
@@ -199,6 +204,64 @@ impl<'backing> TemporaryStore<'backing> {
             .collect()
     }
 
+    // Given the transaction effects, compute the set of config objects that have been
+    // mutated in the transaction.
+    //
+    // Additionally, compute the set of config objects that have been loaded in the current epoch.
+    //
+    // The set of loaded config objects, and mutated config objects are disjoint. All mutated
+    // config objects are transaction inputs. We additionally special-case 0x403 as a config
+    // object, for mutation and loading purposes.
+    fn compute_config_accesses(
+        &self,
+        loaded_per_epoch_config_objects: &BTreeSet<ObjectID>,
+    ) -> (
+        BTreeMap<ObjectID, Option<SequenceNumber>>,
+        BTreeMap<ObjectID, SequenceNumber>,
+    ) {
+        if !self
+            .protocol_config
+            .include_epoch_stable_sequence_number_in_effects()
+        {
+            return (
+                loaded_per_epoch_config_objects
+                    .iter()
+                    .map(|id| (*id, None))
+                    .collect(),
+                BTreeMap::new(),
+            );
+        }
+
+        let mut mutated_configs_at_version = BTreeMap::new();
+        let mut loaded_configs_at_version = BTreeMap::new();
+        for (id, obj) in self.input_objects.iter() {
+            if obj.struct_tag().is_some_and(is_config) || id == &SUI_DENY_LIST_OBJECT_ID {
+                if let Some(((seqno, _), _)) = self.mutable_input_refs.get(id) {
+                    mutated_configs_at_version.insert(*id, *seqno);
+                }
+            }
+        }
+
+        for id in loaded_per_epoch_config_objects.iter() {
+            let seqno = self
+                .store
+                .get_current_epoch_stable_sequence_number(id, self.cur_epoch)
+                .expect("Should panic on storage error")
+                .expect(
```

Comment:
Note the `expect`s here. These are safe since:
1. We should panic on a storage error and not raise any other type of error. 
2. The config (or deny list) can never be deleted, and therefore the object must always exist.

Reply:
These comments should be in the code instead of as github comment